WISE-Community/WISE
# Show node icons in node select menus

Node icons now show up in the node select drop-down menus in the VLE and the AT.

To test:
- View a unit in preview mode and make sure the node icons appear in the node select drop-down.
- Sign in as a student, launch a run, and make sure the node icons appear in the node select drop-down.
- Sign in as a teacher, launch the authoring tool for unit, and make sure the node icons appear in the node select drop-down.

Closes #1845.
